content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -56,14 +56,15 @@ const NewTabPage = React.createClass({
           title="Welcome to new tab"
           body="Firefox will use this space to show your most relevant bookmarks, articles, videos, and pages you've recently visited, so you can get back to them easily."
           label="Identifying your Highlights" />
```

Comment:
Talked this over with @aaronrbenson in IRC.  He suggests leaving the 2s delay before displaying the dialog, and making the code poll every second rather than every three, so I've done that.